synapse/rest/admin/users.py

```diff
@@ -708,3 +719,29 @@ async def on_GET(self, request, user_id):
 
         ret = {"joined_rooms": list(room_ids), "total": len(room_ids)}
         return 200, ret
+
+
+class PushersRestServlet(RestServlet):
```

Comment:
Can you add a docstring to briefly explain what it is and the request/response please?